Yeah, looks like this is the error:
```
[INFO] Rendered /home/vsts/work/1/s/docs/src/main/asciidoc/index.adoc
[INFO] asciidoctor: DEBUG: extension-authors-guide.adoc: line 2354: unknown name for block macro: include
[INFO] asciidoctor: DEBUG: extension-authors-guide.adoc: line 2382: unknown name for block macro: include
[INFO] asciidoctor: WARN: extension-authors-guide.adoc: line 2383: unterminated listing block
[INFO] Rendered /home/vsts/work/1/s/docs/src/main/asciidoc/extension-authors-guide.adoc
[ERROR] asciidoctor: WARN: extension-authors-guide.adoc: line 2383: unterminated listing block
```
which seems unrelated to this change.
I'm away from my workstation to see what this is about.

--- a/extensions/infinispan-embedded/runtime/src/main/java/io/quarkus/infinispan/embedded/runtime/graal/SubstituteJGroups.java
+++ b/extensions/infinispan-embedded/runtime/src/main/java/io/quarkus/infinispan/embedded/runtime/graal/SubstituteJGroups.java
@@ -6,6 +6,7 @@ import org.jgroups.util.UUID;
 
 import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
 public class SubstituteJGroups {
@@ -17,4 +18,32 @@ final class SubstituteUUID {
     // Force it to null - so it can be reinitialized
     @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.Reset)
     protected static volatile SecureRandom numberGenerator;
+}
+
+// DISCARD protocol uses swing classes
+@TargetClass(className = "org.jgroups.protocols.DISCARD")
+final class SubstituteDiscardProtocol {
+
+    @Substitute
+    public void startGui() {
+        // do nothing
+    }
+
+    @Substitute
+    public void stopGui() {
+        // do nothing
+    }
+
+    @Substitute
+    public void start() throws Exception {
+        // should call super.start() but the "super" Protocol.start() does nothing,
+        // so this empty impl is OK
+    }
+
+    @Substitute
+    public void stop() {
+        // should call super.stop() but the "super" Protocol.stop() does nothing,
+        // so this empty impl is OK
+
+    }
 }

